lib/autofs_utils.pm

```diff
@@ -56,23 +56,53 @@ sub check_autofs_service {
     systemctl 'is-active autofs';
 }
 
+=head2 install_service
+
+ install_service();
+
+Install service autofs
+=cut
 sub install_service {
     zypper_call 'in autofs';
 }
 
+=head2 enable_service
+
+ enable_service();
+
+Enable service autofs
+=cut
 sub enable_service {
     systemctl 'enable autofs';
 }
 
+=head2 start_service
+
+ install_service();
+
+Start service autofs
+=cut
 sub start_service {
     systemctl 'start autofs';
 }
 
+=head2 check_service
+
+ check_service();
+
+Check service autofs
+=cut
 sub check_service {
     systemctl 'is-enabled autofs.service';
     systemctl 'is-active autofs';
 }
 
+=head2 configure_service
+
+ configure_service();
+
+Configure service autofs by assert_script_run, check and restart autofs
```

Comment:
Please describe what the argument is about and that it is optional and will default to an empty string.